lib/ui/window.dart

```diff
@@ -225,6 +225,32 @@ abstract class FlutterView {
   ///   applications.
   WindowPadding get padding => viewConfiguration.padding;
 
+  /// Areas of the display that are obstructed by hardware features. This list is
+  /// populated only on Android. If the device has no display features, this list
+  /// is empty.
+  ///
+  /// The space in which the [DisplayFeature.bounds] are defined includes all screens
```

Comment:
This sentence is a bit confusing, not clear what it is implying. The word 'space' seems to be used in multiple contexts here.